prowler/providers/aws/services/fms/fms_accounts_compliant/fms_accounts_compliant.py

```diff
@@ -0,0 +1,27 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.fms.fms_client import fms_client
+
+
+class fms_accounts_compliant(Check):
+    def execute(self):
+        findings = []
+        report = Check_Report_AWS(self.metadata())
+        report.resource_id = "FMS"
+        report.resource_arn = ""
+        report.region = fms_client.region
+        report.status = "PASS"
+        report.status_extended = "FMS disabled or not admin account"
+        if fms_client.fms_admin_account:
+            report.status_extended = "FMS enabled with all compliant accounts"
+            for policy in fms_client.fms_policies:
+                for policy_to_account in policy.compliance_status:
```

Comment:
Here you are iterating over a list, and if one is NONE_COMPLIANT the check is a FAIL, if so why not breaking the loop with a `break` once the first non-compliant policy is detected?